### DISCUSSION TITLE
Is this a fraudulent GitHub copy?

### DISCUSSION THREAD

**Discussion by SavaLione** [community/community#110844] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello,
I'm not sure if this is a good place to ask such a question, but I haven't found a way to contact GitHub DMCA/Security team directly
I recently found this website on Google search: `https://git.histb.com/topics/microcontroller?o=desc&s=updated`

Is this a scam website?

The level 2 domain points to some company that doesn't say that doesn't claim to be affiliated with GitHub
It looks very similarly to official GitHub website and loads some resources from GitHub
Looks very suspicious

Thanks

**Comment by Akash1134** [comment]
We understand the inconvenience caused by these unsolicited phishing notifications and our team is actively working to mitigate them. We want to remind our users to continue to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam) to raise any abusive or suspicious activity.

We would like to bring to our users’ attention to the following:

- Please do not click any links or reply to these notifications. Please [report them](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam).
- [Authorizing an OAuth app](https://docs.github.com/en/apps/oauth-apps/using-oauth-apps/authorizing-oauth-apps) can expose your GitHub account and data to a third party.
- GitHub recommends you periodically [review your authorized OAuth apps](https://docs.github.com/en/apps/oauth-apps/using-oauth-apps/reviewing-your-authorized-oauth-apps).
- This spam activity targets and abuses GitHub’s mention and notification functionality.
- This is a phishing campaign and is not the result of a compromise of GitHub or its systems.

**Reply by SavaLione** [reply]
Thanks
I've read information in the links you provided, but it seems like I can report content that is within GitHub
What should I do if content is outside GitHub?
Are there any GitHub emails or tools that I can use in order to report external link phishing service?

**Reply by natsuki-engr** [reply]
You can choose "Phishing" as a category of your report.
<img width="804" alt="スクリーンショット 2024-06-02 16 55 48" src="https://github.com/community/community/assets/63272932/a5cd27a6-bdbc-4619-b291-69fae8baa856">

**Comment by mrdc** [comment]
> We understand the inconvenience caused by these unsolicited phishing notifications and our team is actively working to mitigate them. 

Hello,
The issue is not solved at the moment: almost 2 months and this phishing site is still alive.
Honestly saying, it doesn’t look like 
> our team is actively working to mitigate them

**Comment by zhgchgli0718** [comment]
It seems to be from a Chinese company: https://histb.com/
```
海纳思系统
一个极低功耗的 Linux 服务器
```

I've already report this site to https://safebrowsing.google.com/safebrowsing/report_phish/?hl=zh-tw

**Comment by EdamAme-x** [comment]
I think that this is website which proxy to github.com

**Reply by EdamAme-x** [reply]
![image](https://github.com/community/community/assets/121654029/b4f172de-e966-4f60-94e1-b017ead74799)
I logged in with tmp account and the login went through. This seems to be a site that proxies.
